src/set_theory/cardinal/ordinal.lean

```diff
@@ -780,6 +780,13 @@ begin
     apply le_max_right }
 end
 
+theorem mk_multiset_eq_max_mk_aleph_0 (α : Type u) [nonempty α] : #(multiset α) = max (#α) ℵ₀ :=
```

Comment:
```suggestion
theorem mk_multiset_of_nonempty (α : Type u) [nonempty α] : #(multiset α) = max (#α) ℵ₀ :=
```
You should also prove the immediate corollary `mk_multiset_of_infinite` and mark it `simp`.

Reply:
And for good measure, you could also prove the trivial `mk_multiset_of_is_empty` (though that one should be provable by `simp` already).

Reply:
I'm not convinced that `mk_multiset_of_nonempty` should be `simp`. Expressions with `max` aren't particularly simpable. I think you should instead mark `mk_multiset_of_infinite` and `mk_multiset_of_finite` (I forgot to mention that one) as `simp`, since presumably for most types you take multisets of, you'll have one of either instance.

Reply:
What's your reasoning for making [cardinal.mk_finsupp_lift_of_infinite](https://leanprover-community.github.io/mathlib_docs/set_theory/cardinal/ordinal.html#cardinal.mk_finsupp_lift_of_infinite) a simp lemma then?
There are simp lemmas cardinal.add_mk_eq_max, cardinal.aleph_add_aleph, cardinal.mul_mk_eq_max, cardinal.aleph_mul_aleph in the same file with max on RHS, and cardinal.max_aleph_eq with max on LHS (and RHS), so I don't think it's a bad idea to have simp lemmas with max on RHS.

I agree it would be nice to have the `finite` version (or better, `countable`). I'll also replace the two `encodable` in this file by the Prop-valued `countable`.

Reply:
simp is able to make use of `max_eq_left` and automatically close the resulting inequality goal using `cardinal.aleph_0_le_mk`. For example, [this](https://github.com/leanprover-community/mathlib/pull/15889/commits/904ef156366575b0dc2cf8c853dfda7ebfba28c0) is now solved by
```
simp only [mv_polynomial.cardinal_mk_eq_max_lift, cardinal.mk_denumerable, cardinal.lift_id, max_eq_right,
  cardinal.aleph_0_le_mk, max_eq_left],
```